package.json

```diff
@@ -5,7 +5,8 @@
   "files": [
     "*.md",
     "docs",
-    "lib"
+    "lib",
+    "src",
```

Comment:
@Kmaschta in order to be able to use the lib with a package reference to the repository